cluster/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -36,10 +36,21 @@ spec:
         - name: AGENT_TOLERATION_KEY
           value: {{ .Values.agent.tolerationKey }}
 {{- end }}
+
 {{- if .Values.agent.flexVolumeDirPath }}
         - name: FLEXVOLUME_DIR_PATH
           value: {{ .Values.agent.flexVolumeDirPath }}
+{{- else }}
+{{- if (semverCompare ">=1.9-0" .Capabilities.KubeVersion.GitVersion) }}
+        - name: FLEXVOLUME_DIR_PATH
+          value: "/var/lib/kubelet/volumeplugins/"
+{{- else }}
+        - name: FLEXVOLUME_DIR_PATH
+          value: "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/"
```

Comment:
our docs don't mention setting this default. It would be nice either to mention this default in the docs, or remove it from here for consistency.